src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+const initialState = {
+  books: [],
+};
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+const categoriesSlice = (state = initialState, action) => {
+  if (action.type === CHECK_STATUS) {
+    return 'Under construction';
+  }
+  return state;
+};
+
+export default categoriesSlice;
```

Comment:
- [ ] Please kindly implement this using redux toolkit (RTK) according to the project requirement, it seems like you're using the old method. ℹ️ 

<img width="791" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230048704-13d254c9-90e9-407e-b7d5-19c1464cb418.png">